tests/unit/test_routes.py

```diff
@@ -283,6 +283,13 @@ def add_policy(name, filename):
         pretend.call("ses.hook", "/_/ses-hook/", domain=warehouse),
         pretend.call("rss.updates", "/rss/updates.xml", domain=warehouse),
         pretend.call("rss.packages", "/rss/packages.xml", domain=warehouse),
+        pretend.call(
+            "rss.user_updates",
+            "/user/{username}/updates.xml",
```

Comment:
Looking at #7013, I think this should probably be `/rss/user/{username}/something.xml`, where `something` is distinctly one of the two types of feeds discussed in #4603.